tutorials/classify-maizeland-ng/index.md

```diff
@@ -0,0 +1,235 @@
+var validated_rf = testing.classify(trained_rf);
+
+// Calculate confusion matrix to generate validation accuracy for CART
+var testAccuracy = validated.errorMatrix('class', 'classification');
+print('Validation error matrix: ', testAccuracy);
+print('Validation overall accuracy: ', testAccuracy.accuracy());
+
+// Calculate confusion matrix to generate validation accuracy for CART
+var testAccuracy_rf = validated_rf.errorMatrix('class', 'classification');
+print('Validation error matrix: ', testAccuracy_rf);
+print('Validation overall accuracy: ', testAccuracy_rf.accuracy());
+
+//Draw the "aoi" layer on top of the classified grid for visualization. You can tweak the opacity parameter or turn off the "aoi" layer to see the layer beneath more clearly
+Map.addLayer(aoi);   
+```
+
+## 3. Calculate area and Export the output
+With the binary classification completed, you can now export the classified imagery to google bucket drive (or any other compatible storage) for further analysis. Check the export resolution parameter (called "scale") and adjust accordingly to control output file size, if neccessary. The larger the scale, the lighter the file size. The maxPixels parameter sets an upper boundary on th enumber of pixels allowable for export to avoid export of large file or prolonged file creation time. Calculate the area for each landcover class by applying ee.Image.pixelArea on the classifed imagery and assign to the variable areaImage. By passing on the new variable to the Image reducing function, constrained by the aoi boundary geometry and specifying other parameters (per below), the area for both classes are generated in squeare meters. 
+
+```js
+//Exporting to google bucket drive; If you use any other type of staorage solution, set the command as appropriate
+  Export.image.toDrive({
+ image: classified,
+  description: 'Maizeland_Classified',
+  scale: 20,
+  region: aoi,
+  maxPixels: 100000000000,
+});
+
+//To calculate Maize Area in square meters
+var areaImage = ee.Image.pixelArea().addBands(
+      classified_rf);
+      
+var areas = areaImage.reduceRegion({
+      reducer: ee.Reducer.sum().group({
+      groupField: 1,
+      groupName: 'class',
+    }),
+    geometry: aoi.geometry(),
+    scale: 500,
+    maxPixels: 1e10
+    }); 
+ print (areas);
+```
+
+## 4.Final notes
+Although this tutorial offers a template for users, further tweaks can possibly improve the results.  The results (in the Console tab) shows that RF outperformed CART in the validation mode (RF accuracy = 73.4%; CART accuracy = 69%). Also, the maize area estimated was 19.09*1e10 sq meter (i.e. ~19 million ha). Is is probable that this is over/under estimated, so further area-based validation may be necessary to validate the estimate. In any case, this analytical tool (and tutorial) can support rapid generation of national cropland area estimate that is scalable (across, regions, local governments/districts, wards or villages) based on delineated boundaries. 
```

Comment:
Please also fix the following lint errors (most some already mentioned above):
```
  70:1   error  Expected space or tab after '//' in comment            spaced-comment
   71:43  error  Trailing spaces not allowed                            no-trailing-spaces
   74:47  error  Trailing spaces not allowed                            no-trailing-spaces
   76:1   error  Expected space or tab after '//' in comment            spaced-comment
   79:1   error  Expected space or tab after '//' in comment            spaced-comment
   80:43  error  Trailing spaces not allowed                            no-trailing-spaces
   90:1   error  Expected space or tab after '//' in comment            spaced-comment
   91:31  error  Missing space before value for key 'color'             key-spacing
   92:30  error  Missing space before value for key 'color'             key-spacing
   98:1   error  Expected space or tab after '//' in comment            spaced-comment
   99:29  error  Strings must use singlequote                           quotes
  105:1   error  Expected space or tab after '//' in comment            spaced-comment
  114:1   error  Expected space or tab after '//' in comment            spaced-comment
  119:1   error  This line has a length of 184. Maximum allowed is 100  max-len
  119:1   error  Expected space or tab after '//' in comment            spaced-comment
  127:1   error  Expected space or tab after '//' in comment            spaced-comment
  128:18  error  A space is required after ','                          comma-spacing
  128:23  error  A space is required after ','                          comma-spacing
  128:28  error  A space is required after ','                          comma-spacing
  128:33  error  A space is required after ','                          comma-spacing
  128:38  error  A space is required after ','                          comma-spacing
  128:75  error  A space is required after ','                          comma-spacing
  130:5   error  Identifier 'image_cl' is not in camel case             camelcase
  178:7   error  Parsing error: Unexpected token assess
  209:1   error  This line has a length of 114. Maximum allowed is 100  max-len
  209:1   error  Expected space or tab after '//' in comment            spaced-comment
  218:1   error  Expected space or tab after '//' in comment            spaced-comment
  221:1   error  Trailing spaces not allowed                            no-trailing-spaces
  230:8   error  Trailing spaces not allowed                            no-trailing-spaces
  231:2   error  Unexpected newline between function name and paren     func-call-spacing
```

Reply:
done